schema/home/results.js

```diff
@@ -18,8 +18,8 @@ const RESULTS_SIZE = 20;
 
 const moduleResults = {
   active_bids: () => [],
-  iconic_artists: () => {
-    return iconicArtists().then((artists) => {
+  trending_artists: () => {
```

Comment:
Ah, so you do have a rail with these works, I wasn’t able to find it in the designs I was looking at on Wake.

So, yeah, it is breaking, but I would also like to know how ‘iconic artists’ is defined in this context at all, because these appear to be ‘trending artists’ (if I understand the Delta call correctly).

From [what I know](https://github.com/artsy/eigen/issues/1419), ‘iconic artists’ are those that have the most inquiries made for their works:

> This list is based on the top inquired artists on Artsy.

The inquiry data does exist in Delta, but as far as I can tell there still needs to be a Gravity endpoint that fetches that data from Delta and collects the actual artists for those hits.